src/key-managment/AwsKeyManagement.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using Amazon.KeyManagementService;
+using Amazon.KeyManagementService.Model;
+
+namespace Kamus.KeyManagement
+{
+    public class AwsKeyManagement : IKeyManagement
+    {
+        private readonly IAmazonKeyManagementService mAmazonKeyManagementService;
+        private readonly SymmetricKeyManagement mSymmetricKeyManagement;
+
+        public AwsKeyManagement(IAmazonKeyManagementService amazonKeyManagementService, SymmetricKeyManagement symmetricKeyManagement)
+        {
+            mAmazonKeyManagementService = amazonKeyManagementService;
+            mSymmetricKeyManagement = symmetricKeyManagement;
+        }
+
+        public async Task<string> Encrypt(string data, string serviceAccountId, bool createKeyIfMissing = true)
+        {
+            var masterKeyAlias = $"alias/kamus/{KeyIdCreator.Create(serviceAccountId)}";
+            (var encryptionKey, var encryptedDataKey ) = await GenerateEncryptionKey(masterKeyAlias);
```

Comment:
```suggestion
            (var dataKey, var encryptedDataKey ) = await GenerateDataKey(masterKeyAlias);
```
It's a bit neatpicking, but naming is important :)